api/src/main/java/pl/kolendateam/dadcard/abilitys/entity/Abilitys.java

```diff
@@ -56,5 +56,19 @@ public Abilitys addRaceAbilitys(Abilitys jsonObjectAbilitys, Abilitys abilitys)
         
         return abilitys;
     }
+
+    public int checkPrerequisiteAb(Abilitys ab) {
```

Comment:
jak check to powinno zwracac boolean

Reply:
tak zmienię, zacząłem pisać bo myślałem że będzie potrzebny też inne numery niż 1 i 0. Ale wystarczyły 2.